lib/cucumber/listener/rerun_formatter.js

```diff
@@ -0,0 +1,36 @@
+function RerunFormatter(options) {
+
+	var Cucumber = require('../../cucumber');
```

Comment:
Please convert all tabs to spaces. That should fix the formatting issues seen here